toolkit/scripts/pkggen.mk

```diff
@@ -51,12 +50,26 @@ clean-cache:
 	$(SCRIPTS_DIR)/safeunmount.sh "$(cache_working_dir)" && \
 	rm -rf $(cache_working_dir)
 
+# Optionally generate a summary of any blocked packages after a build.
+analyze-built-graph: $(go-graphanalytics)
+	if [ ! -f $(build_file) ]; then \
```

Comment:
>if [ ! -f $(build_file) ]; then \ [](start = 1, length = 33)

Shouldn't this condition be the other way around? Right now it looks like we're providing `$(built_file)` as input only if it's **not** a file.